examples/parsons_civis_twilio_example.py

```diff
@@ -0,0 +1,44 @@
+# This script is an example of using the parsons library to import data from
+# Twilio using the parsons Twilio connector, and then upload the resulting
+# table to Platform using the parsons Civis connector.
+from parsons.civis.civisclient import CivisClient
+from parsons.twilio.twilio import Twilio
+
+# Step 1: Open a new Python script in Platform, and paste this entire file
+#         into the script body.
+#         TODO: this would require adding parsons to datascience-python
+
+# Step 2: Set CIVIS_DATABASE and CIVIS_TABLE variables.
+#         CIVIS_TABLE should be in "schema.table" form
+#         See list of databases: platform.civisanalytics.com/spa/remote_hosts
+
+# Step 3: Set environment variables necessary for the clients:
+#         CIVIS_API_KEY: Civis API key
+#         TWILIO_ACCOUNT_SID: Twilio Account SID
+#         TWILIO_AUTH_TOKEN: Twilio Auth Token
```

Comment:
Should the auth token be stored in a credential? That's going to look more like an environment variable with `_PASSWORD` at the end?